types/google-map-react/index.d.ts

```diff
@@ -66,30 +66,191 @@ declare namespace googleMapReact {
     }
 
     interface Maps {
-        Animation: any;
-        ControlPosition: any;
-        MapTypeControlStyle: any;
-        MapTypeId: any;
-        NavigationControlStyle: any;
-        ScaleControlStyle: any;
-        StrokePosition: any;
-        SymbolPath: any;
-        ZoomControlStyle: any;
-        DirectionsStatus: any;
-        DirectionsTravelMode: any;
-        DirectionsUnitSystem: any;
-        DistanceMatrixStatus: any;
-        DistanceMatrixElementStatus: any;
-        ElevationStatus: any;
-        GeocoderLocationType: any;
-        GeocoderStatus: any;
-        KmlLayerStats: any;
-        MaxZoomStatus: any;
-        StreetViewStatus: any;
-        TransitMode: any;
-        TransitRoutePreference: any;
-        TravelMode: any;
-        UnitSystem: any;
+        Animation: {
+            BOUNCE: number;
+            DROP: number;
+            Uz: number;
+            Qz: number;
+        };
+        ControlPosition: {
+            TOP_LEFT: number;
+            TOP_CENTER: number;
+            TOP: number;
+            TOP_RIGHT: number;
+            LEFT_CENTER: number;
+            LEFT_TOP: number;
+            LEFT: number;
+            LEFT_BOTTOM: number;
+            RIGHT_TOP: number;
+            RIGHT: number;
+            RIGHT_CENTER: number;
+            RIGHT_BOTTOM: number;
+            BOTTOM_LEFT: number;
+            BOTTOM_CENTER: number;
+            BOTTOM: number;
+            BOTTOM_RIGHT: number;
+            CENTER: number;
+        };
+        DirectionsStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            MAX_WAYPOINTS_EXCEEDED: string;
+            NOT_FOUND: string;
+        };
+        DirectionsTravelMode: {
+            DRIVING: string;
+            WALKING: string;
+            BICYCLING: string;
+            TRANSIT: string;
+            TWO_WHEELER: string;
+        };
```

Comment:
Could not find `DirectionsTravelMode` in [the documentation](https://developers.google.com/maps/documentation/javascript/reference) (it seems to be the same as `TravelMode` though). Maybe it should be deprecated with a message telling users to use `TravelMode`, wdyt ?

Reply:
Please see [main comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60329#issuecomment-1126798899) for response RE where this was located. 

In terms of being deprecated, it seems that these were renamed back in 2011 but still persist for backwards compatability (see [Release Notes](https://developers.google.com/maps/documentation/javascript/releases?hl=en#april-14,-2011)) - quoted below:

> Renamed DirectionsTravelMode and DirectionsUnitSystem to TravelMode and UnitSystem (old names remain backwards compatible) 

As such it likely makes sense to mark as deprecated with the message - I'll make that change.